### PR TITLE
Page transitions.

### PR DESCRIPTION
**Pull request by mfduar8766**
@smuntean @iangornall

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -7,6 +7,10 @@
   border: solid 0.3em var(--dk-blue);
 }
 
+.hide-form {
```
Comment (sarathrasher) [inline public/css/login.css]: We have a "hidden" class already. It's in the main css file

File: public/js/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+let playBtn = document.querySelector(".nav-link-play");
+let instructions = document.querySelector(".instructions");
+let game = document.querySelector(".game");
+let instructionsStartGame = document.querySelector(".start-button");
+let statsBtn = document.querySelector(".nav-link-stat");
+
+let showInstructions = () => {
+  instructions.classList.remove("hidden");
+  let form = document.querySelector(".login-form");
+  form.classList.add("hide-form");
```
Comment (sarathrasher) [inline public/js/main.js]: I think you want to hide the form before you show the instructions, and why not just add the hidden class that we already have to this?

File: public/js/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+};
+
+let showGame = () => {
+  game.classList.remove("hidden");
+  instructions.classList.add("hidden");
```
Comment (sarathrasher) [inline public/js/main.js]: Again, I think you want to hide the instructions before you start the game

File: public/js/main.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+let loginBtn = document.querySelector(".login-button");
+let createBtn = document.querySelector(".create-button");
+let playBtn = document.querySelector(".nav-link-play");
+let instructions = document.querySelector(".instructions");
+let game = document.querySelector(".game");
+let instructionsStartGame = document.querySelector(".start-button");
+let statsBtn = document.querySelector(".nav-link-stat");
```
Comment (sarathrasher) [inline public/js/main.js]: It's actually '.nav-link-stats' with an s

**Review by sarathrasher — DISMISSED** [review]
See comments.

File: public/js/main.js (hunk 2 above)
Comment (mfduar8766) [reply public/js/main.js]: The code I wrote hides the form when you fill it out. It then shows the instructions. Then when you click the play button on the instructions it hidesvthat too.

Comment (mfduar8766) [reply public/js/main.js]: The code I wrote hides the form when you fill it out. It then shows the instructions. Then when you click the play button on the instructions it hides the instructions and shows the game.

File: public/css/login.css (hunk 1 above)
Comment (mfduar8766) [reply public/css/login.css]: Cool. I used it

**Review by iangornall — APPROVED** [review]
👍

**Pull request merged**